### PR TITLE
[res] TF example for BatchNorm, FusedBatchNorm

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce TensorFlowPythonExamples for batch_normalization and fused_batch_norm

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>
ONE-DCO-1.0-Signed-off-by: Alexander Efimov a.efimov@samsung.com

### CODE REVIEW

**Comment by seanshpark** [comment]
Code from https://github.com/Samsung/ONE/issues/1167#issuecomment-635999572 examples.zip

**Comment by seanshpark** [comment]
@binarman , PTAL

**Comment by binarman** [comment]
Thank you!
